src/main/java/mate/academy/spring/validator/EmailValidator.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.spring.validator;
+
+import java.util.regex.Pattern;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EmailValidator implements ConstraintValidator<Email, String> {
+    private static final String EMAIL_PATTERN = "^[A-Z0-9._%+-]+@[A-Z0-9.-]+\\\\.[A-Z]{2,6}$";
+
+    @Override
+    public boolean isValid(String email, ConstraintValidatorContext constraintValidatorContext) {
+        return email != null && Pattern.compile(EMAIL_PATTERN).matcher(email).matches();
```

Comment:
It is a good practice to compile your pattern only once in such services. It is better to create a constant of a type Pattern but not String
(I had the same problem)